pkgs/tools/backup/gickup/default.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "gickup";
+  version = "0.10.17";
+
+  src = fetchFromGitHub {
+    owner = "cooperspencer";
+    repo = "gickup";
+    rev = "v${version}";
+    sha256 = "sha256-tiQmb7bBWb99k23lS+d+YR14y4YeYPWqccl/2DLv7Dk=";
```

Nowadays, using the SRI hash is the default.

```suggestion
    hash = "sha256-tiQmb7bBWb99k23lS+d+YR14y4YeYPWqccl/2DLv7Dk=";
```